covid_xprize/examples/predictors/linear/Example-Train-Linear-Rollout-Model.ipynb

```diff
@@ -127,7 +127,7 @@
    "outputs": [],
    "source": [
     "# Add new cases column\n",
-    "df['NewCases'] = df.groupby('GeoID').ConfirmedCases.diff().fillna(0)"
+    "df['NewCases'] = df.groupby('GeoID', group_keys=False).ConfirmedCases.diff().fillna(0)"
```

Comment:
Same fix everywhere: specified `group_keys=False` for each groupby to preserve existing behavior.